pom.xml

```diff
@@ -2774,6 +2774,10 @@
             <groupId>com.google.http-client</groupId>
             <artifactId>google-http-client</artifactId>
           </exclusion>
+          <exclusion>
```

Comment:
Why does upgrading `tink` require adding this `exclusion`?

Reply:
Cause that `tink` upgrade `protobuf-java` to 4.27.0 in version 1.14.0, but it's not compatible with 3.25.1. (related protobuf issues: https://github.com/protocolbuffers/protobuf/issues/16452, https://github.com/protocolbuffers/protobuf/issues/17247)

And I notice that there are some users in `tink` community also facing this problem, let's wait for the community to provide a solution(https://github.com/tink-crypto/tink-java/issues/31 and https://github.com/tink-crypto/tink-java/pull/38). @LuciferYang

Reply:
If that's the case, I suggest skipping this version.

Reply:
Thanks for the suggestion. I will close this PR first and keep watching on this protobuf realted issue in the future.